README.md

```diff
@@ -70,69 +76,66 @@ Use `grunt serve` for development.
 
 ## Support/Help
 
-- IRC: irc.hackint.org #freifunkRGB
+- IRC: irc.hackint.org #freifunkRGB or development-channel #meshviewer
 - Feel free to open an issue for a problem or a idea.
```

Comment:
add a link to `issue`:

    Feel free to open an [issue](https://github.com/ffrgb/meshviewer/issues/new) for a problem or an idea.